### PR TITLE
Update to freedesktop 22.08

### PR DESCRIPTION
**Pull request by Erick555**
Also update various components

**Need to be merged in 22.08 branch**

### CODE REVIEW

File: org.mozilla.firefox.BaseApp.json [hunk 1]
```diff
@@ -47,8 +52,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.9.tar.xz",
-                    "sha256": "66c0517ed16df7af258e83208faaf5069727dfd66995c4bbc51c16954d674761"
+                    "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.12.tar.xz",
```
Comment (Lctrs) [inline org.mozilla.firefox.BaseApp.json]: Could be updated to 0.8.1 I think

**Comment by hfiguiere** [comment]
openssl 3 failure?

**Comment by hfiguiere** [comment]
nvm, builds out of order.

File: org.mozilla.firefox.BaseApp.json (hunk 1 above)
Comment (Erick555) [reply org.mozilla.firefox.BaseApp.json]: Sure, I  missed it

**Comment by Erick555** [comment]
> nvm, builds out of order.

Yeah for some reason flathub builds doesn't finish immediately after failure but wait until timeout is hit.

**Comment by barthalion** [comment]
Done, thanks!

**Pull request closed without merging**